### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [ ] ~Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)~
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [ ] ~Confirming a section header is included for Quickstart UI rendering (only for transform repos)~
    - [ ] ~Renaming `**model` to `**table**` (only for transform repos)~
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,60 +11,60 @@
...
 - These tables will flow up to replicate SAP extractor reports that are provided in our transformation package, while not applying any renaming to the fields.
 - These staging tables clean, test, and prepare your SAP data from [Fivetran's SAP connectors, like LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements), [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent) or [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) for analysis by doing the following:
-  - Name columns for consistency across all packages and for easier analysis
-  - Adds freshness tests to source data
-  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Name columns for consistency across all packages and for easier analysis
+- Adds freshness tests to source data
+- Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
```
Comment (fivetran-joemarkiewicz) [inline README.md]:
```suggestion
  - Name columns for consistency across all packages and for easier analysis
  - Adds freshness tests to source data
  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
```

File: README.md [hunk 2]
```diff
@@ -11,60 +11,60 @@
...
+## How do I use the dbt package?
+### Step 1: Prerequisites
 To use this dbt package, you must have the following:
 - At least one Fivetran of the following SAP connectors:
-   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
-   - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
-   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
+- [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
+- [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
+- [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)
```
Comment (fivetran-joemarkiewicz) [inline README.md]:
```suggestion
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)
```

File: README.md [hunk 3]
```diff
@@ -11,60 +11,60 @@
...
+- mara
+- pa0000
+- pa0001
+- pa0007
+- pa0008
+- pa0031
+- ska1
+- t001
+- t503
+- t880
```
Comment (fivetran-joemarkiewicz) [inline README.md]:
```suggestion
   - bkpf
   - bseg
   - faglflexa
   - faglflext
   - kna1
   - lfa1
   - mara
   - pa0000
   - pa0001
   - pa0007
   - pa0008
   - pa0031
   - ska1
   - t001
   - t503
   - t880
```

**Comment by fivetran-joemarkiewicz** [comment]
I'm aware the integration tests aren't passing. However, I believe this is an issue with the underlying code and should be addressed in a separate PR.

**Review by fivetran-catfritz — APPROVED** [review]
lgtm

**Pull request merged**